.circleci/config.yml

```diff
@@ -48,30 +48,24 @@ jobs:
       - image: circleci/python:3.6
         environment:
           TOXENV: lint
-  py36-core:
+  py36-test:
     <<: *common
     docker:
       - image: circleci/python:3.6
         environment:
-          TOXENV: py36-core
-  py37-core:
+          TOXENV: py36-test
+  py37-test:
     <<: *common
     docker:
       - image: circleci/python:3.7
         environment:
-          TOXENV: py37-core
-  pypy3-core:
-    <<: *common
-    docker:
-      - image: pypy
-        environment:
-          TOXENV: pypy3-core
+          TOXENV: py37-test
+
 workflows:
   version: 2
   test:
     jobs:
       - docs
       - lint
-      - py36-core
-      - py37-core
-      - pypy3-core
```

Comment:
If we can get pypy3 support, it's nice to have this in to prevent accidentally losing support. Since it's not blocking this PR (not running Circle yet), I'm :+1: for leaving it in for now. (especially the test definition, even if it's removed from the `workflows`)

... but if you know pypy3 support is not possible currently, and would take a lot of work, then no objection to dropping it now.

Reply:
good point, i don't know about `pypy3` support and i'm actually inclined to just revert changes to this file